src/components/PeoplePage.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+
+import { getPeople } from '../api/people';
+
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
```

Comment:
You could declare this function outside the `useEffect` and just call it here. Also, instead of passing `people` in props to the `PeopleTable` component, you could fetch people right directly in that component